tests/e2e/runner/__tests__/base.test.ts

```diff
@@ -1,78 +1,42 @@
-import fs from "fs";
 import { describe, test, expect } from "@jest/globals";
 import {
-  createClient,
-  dedupExchange,
-  cacheExchange,
-  fetchExchange,
-  Client,
-} from "@urql/core";
-import { retryExchange } from "@urql/exchange-retry";
-import {
-  GetContractsDocument,
-  GetContractsQuery,
-  GetContractsQueryVariables,
-  GetCurrentBlockNumberDocument,
-  GetCurrentBlockNumberQuery,
+  GetTargetContractsDocument,
+  GetTargetContractsQuery,
+  GetTargetContractsQueryVariables,
 } from "../generated/graphql";
-import { SubgraphConfig } from "./subgraph-config";
-import fetch from "cross-fetch";
-import { ethers } from "ethers";
-
-const createSubgraphClient = (): Client => {
-  const client = createClient({
-    url: process.env.SUBGRAPH_GRAPHQL_URL as string,
-    fetch: fetch as any,
-    exchanges: [
-      dedupExchange,
-      cacheExchange,
-      retryExchange({
-        maxNumberAttempts: 5,
-        retryIf: (error) => !!error,
-      }),
-      fetchExchange,
-    ],
-    requestPolicy: "network-only",
-  });
-
-  return client;
-};
+import {
+  createSubgraphClient,
+  waitUntilSubgraphIsSynced,
+  getSubgraphConfig,
+} from "./utils/helpers";
 
-const contracts = JSON.parse(
-  fs.readFileSync("/usr/runner/shared/test-config.json", "utf-8")
-) as SubgraphConfig;
+// waiting for subgraph to sync can take longer than the default 5s timeout
+jest.setTimeout(30 * 1000);
```

Comment:
Kinda surprised by this since honestly, since everything's on a local network would expect things to sync very quickly probably worth looking into in the future.

Reply:
Ah, I added this at one point, but I think I realized later the delay was being caused by something else that I fixed. I've removed these timeout overrides and should be fine without them!

Reply:
@yoshiwarab - I removed this, then actually did encounter a failure in `minter-filter-v2.test.ts` on one of the tests that performs a bunch of operations, waits for sync, queries subgraph, performs more operations, waits for sync, queries, and so on...
So I did bump up that file's timeout to 15s. Happy to chat more on this if you want!